core/jazz_cognito-admin-authorizer/index.js

```diff
@@ -411,7 +411,7 @@ module.exports.handler = (event, context, cb) => {
        *
        * @method denyAllMethods
        */
-      denyAllMethods: () => {
+      denyAllMethods: function () {
```

Comment:
why?

Reply:
It was throwing TypeError with arrow function. May be it is due to the context attached to "this".

Reply:
It was throwing TypeError with arrow function. May be it is due to the context attached to "this".